Temp.java

```diff
@@ -0,0 +1,49 @@
+import java.util.*;
+
+public class Timedifference {
+    int sec,hr,min;
+    public Timedifference(int hr,int min, int sec){
+        this.hr = hr;
+        this.min = min;
+        this.sec = sec;
+    }
+    public static Timedifference Dif(Timedifference T1, Timedifference T2){
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.